developer/src/kmc-ldml/build.sh

```diff
@@ -34,17 +36,34 @@ builder_describe_outputs \
 builder_parse "$@"
 
 function do_clean() {
-  rm -rf ./build/ ./tsconfig.tsbuildinfo
+  rm -rf ./build/ ./tsconfig.tsbuildinfo ./src/util/abnf/*.pegjs
 }
 
 function do_configure() {
   verify_npm_setup
+  do_build_abnf
 }
 
 function do_build() {
   npm run build
 }
 
+function do_build_abnf() {
+  ABNF_SRC="$KEYMAN_ROOT/resources/standards-data/ldml-keyboards/$LDML_CLDR_VERSION_LATEST/abnf"
+  for file in ${ABNF_SRC}/*.abnf; do
+    base=$(basename "$file" .abnf)
+    peg="$base.pegjs"
+    outfile="./src/util/abnf/$peg"
+    outjs="./src/util/abnf/$base.js"
+    if [ ! -f "$outfile" ]; then
+      printf "${COLOR_GREY}abnf_gen ${COLOR_PURPLE}${base}.abnf -> ${peg}${COLOR_RESET}"
+      npx -p abnf abnf_gen  "$file" -o "$outfile"
+      printf "${COLOR_GREY}abnf_gen ${COLOR_PURPLE}${peg} -> ${base}.js${COLOR_RESET}"
+      npx peggy "$outfile" -o "$outjs" --format es --dts
```

Comment:
I am not a big fan of npx. Can we call the tools more directly, either with ./node_modules/.bin/abnf or `npm run abnf` (and corresponding script in package.json).

Reply:
I'll just change this to .bin